Server/Server.Core/Server.Core.Static/SettingsManager.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "Settings.hpp"
+
+namespace Server::Builder
+{
+class SettingsManager
+{
+public:
+    SettingsManager() = default;
+
+    explicit SettingsManager(std::unique_ptr<Settings> settings) : _settings(std::move(settings)) {}
+    
+    void Reset(std::unique_ptr<Settings> settings) { _settings.reset(settings.release()); }
+
+    /**
+    * @brief Generates connection string
+    * @details In this case, for now, connection string to Postgre DB only
+    */
+    std::string GetConnectionOptions() const
```

Comment:
The names of the functions do not agree with the naming convention

Reply:
Done